.jenkins/short-perf-test-cpu.sh

```diff
@@ -7,41 +7,48 @@ cd .jenkins/perf_test
 
 echo "Running CPU perf test for PyTorch..."
 
-# Get last master commit hash
-export PYTORCH_COMMIT_ID=$(git log --format="%H" -n 1)
+pip install awscli
 
-# Get baseline file from https://github.com/yf225/perf-tests
-if [ -f /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json ]; then
-    cp /var/lib/jenkins/host-workspace/perf_test_numbers_cpu.json perf_test_numbers_cpu.json
-else
-    curl https://raw.githubusercontent.com/yf225/perf-tests/master/perf_test_numbers_cpu.json -O
+export AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+export AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+
+if [[ "$COMMIT_SOURCE" == *master* ]]; then
+    # Get current master commit hash
+    export MASTER_COMMIT_ID=$(git log --format="%H" -n 1)
 fi
 
-if [[ "$GIT_COMMIT" == *origin/master* ]]; then
+# Get baseline file from ossci-perf-test S3 bucket
+aws s3 cp s3://ossci-perf-test/pytorch/cpu_runtime/LATEST_TESTED_COMMIT LATEST_TESTED_COMMIT
```

Comment:
I want you to set a sufficiently high multipart threshold so this is not a multipart read. Background at: https://github.com/aws/aws-cli/issues/2321